common/test/acceptance/pages/lms/problem.py

```diff
@@ -212,7 +211,14 @@ def is_focus_on_problem_meta(self):
         """
         Check for focus problem meta.
         """
-        return is_focused_on_element(self.browser, '.problem-header')
+        return self.q(css='.problem-header').focused
+
+    def wait_for_focus_on_problem_meta(self):
+        """
+        Waits for focus on Problem Meta section
+        """
+        self.wait_for(promise_check_func=self.is_focus_on_problem_meta,
```

Comment:
Nit: the standard way to wrap this would be:

    self.wait_for(
        promise_check_func=self.is_focus_on_problem_meta,
        description='Waiting for focus on Problem Meta section',
    )